common/utils/src/main/resources/error/error-conditions.json

```
@@ -5141,6 +5141,11 @@
         "message" : [
           "TRIM specifier in the collation."
         ]
+      },
+      "UPDATE_COLUMN_NULLABILITY" : {
+        "message" : [
+          "UpdateColumnNullability for MySQL and MS SQL Server."
```

I think SQL users are not aware of Spark's internal methods. Let's split the words:
```suggestion
          "Update column nullability for MySQL and MS SQL Server."
```

---

Updated. Looks better 👍